apps/files/src/components/FileLink.vue

```diff
@@ -27,6 +27,9 @@
         <FileLinkForm v-if="linkId === link.id" class="uk-margin-top" v-bind:params="params" :context="'edit'" :linkId="linkId"/>
       </li>
     </transition-group>
+    <p v-if="$_links.length === 0" class="uk-text-meta" v-translate>
+      Links can be shared with collaborators outside of ownCloud.
```

Comment:
Owncloud has to be replaced by the Brand Name from the configuration state

Reply:
If you refer to (public) links here, the term "Collaborators" is misleading as it refers to internal users in the new concept. Maybe go for "Links can be shared with **external** collaborators." to make this more clear or find a different wording for "Collaborators" (I don't have a good idea right now). Just "Collaborators" should only be used when referring to private links.

Reply:
I'm not very good at phrasing things but in essence … we need a description that clearly indicates **what** a link is and **when** to use it over a local share.

Reply:
I updated as @pmaier1 suggested.